### PR TITLE
Add datadog started metric

### PR DESCRIPTION
**Pull request by nmuesch**
### What does this PR do?

Adds in a new metric `datadog.agent.running`

### Motivation

Showcase whether an Agent is running, similar to the service check.

### Additional Notes

This also allows us to identify this as an Agent host and showcase the dog bone icon in the Infrastructure List

### CODE REVIEW

**Review by gmmeyer — COMMENTED** [review]
I don't think we want to send this with every series we flush. We could be sending this dozens of times per collection run. It'll pound us with metrics we don't need.

Instead, it should be sent on a timer like checks and metadata. It could just be a simple goroutine that runs in the background with a ticker. 

We do something similar to this after every check run, you can see this here: https://github.com/DataDog/datadog-agent/blob/eca175e4010af6dbccc930cb6db9caa8e6067452/pkg/collector/runner/runner.go#L260-L284

The easiest alternative would be to send it after every check run as we do above, I still think that's a bit overkill but it's a lot better.

**Comment by olivielpeau** [comment]
@gmmeyer this change would actually send the metric `datadog.agent.running` once per main aggregator flush, i.e. by default once every 15 seconds. That sounds reasonable to me, it only adds one metric on one context per agent, that's sent every 15 seconds.

File: pkg/aggregator/aggregator.go [hunk 1]
```diff
@@ -246,6 +246,16 @@ func (agg *BufferedAggregator) addServiceCheck(sc metrics.ServiceCheck) {
 	agg.serviceChecks = append(agg.serviceChecks, &sc)
 }
 
+func (agg *BufferedAggregator) addSerie(serie metrics.Serie) metrics.Series {
```
Comment (olivielpeau) [inline pkg/aggregator/aggregator.go]: don't think it makes sense to have an `addSerie` function here given how the series are aggregated

**Review by olivielpeau — COMMENTED** [review]
Thanks for taking care of this!

Made a comment, the RC.1 freeze is imminent so going to address it myself

**Comment by olivielpeau** [comment]
Going to go ahead and merge this for RC.1.

If anything doesn't look right in my changes let me know and I'll address

**Comment by gmmeyer** [comment]
Oh! That makes sense! Okay!

**Pull request merged**